habitat/tasks/nav/nav.py

```diff
@@ -509,6 +509,41 @@ def update_metric(
         )
 
 
+@registry.register_measure
+class SoftSPL(SPL):
+    r"""Soft SPL
+
+    Similar to SPL with a relaxed soft-success criteria. Instead of a boolean
+    success is now calculated as 1 - (ratio of distance covered to target).
+    """
+
+    def _get_uuid(self, *args: Any, **kwargs: Any):
```

Comment:
@abhiskk, looks like `reset` method was lost. Worth to add minimal valuable test so we could caught changes like this.

Reply:
it should require dependency metric:
```
     task.measurements.check_measure_dependencies( 
         self.uuid, [DistanceToGoal.cls_uuid] 
     ) 
```

Reply:
I think since it inherits from SPL (and thus gets its reset method), the right thing will already happen?

Reply:
Yes, because it inherits reset functionality will remain the same.

Reply:
@abhiskk, SPL depends on Success metrics, while SoftSPL isn't.